test/test_assembly.py

```diff
@@ -15,10 +15,13 @@
 from __future__ import print_function, division, absolute_import
 
 from isovar.variant_reads import reads_supporting_variant
-from isovar.variant_sequences import initial_variant_sequences_from_reads
+from isovar.variant_sequences import (
+    initial_variant_sequences_from_reads,
+    VariantSequence
```

Comment:
why import VariantSequence here? Doesn't seem like you use it in any of the other changes to this file. Also, seems unrelated to this PR - ideally this and other unrelated test file changes should be in a different branch/PR - okay for now but try not to add any more here :)